doc/api/events.md

```diff
@@ -1282,9 +1282,11 @@ This is not used in Node.js and is provided purely for completeness.
 added: v14.5.0
 -->
 
-* Type: {boolean} Always returns `false`.
+* Type: {boolean} Returns `false` for user constructed events and `true`
```

Comment:
```suggestion
* Type: {boolean} Value is `false` for user-constructed events and `true`
```

Reply:
All other properties (like `event.returnValue`) just use "True if..." so I will change to that?

Reply:
Sure, that works.